test/micro/org/openjdk/bench/java/io/DataInputStreamTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2022, Red Hat Inc. All rights reserved.
+ * Copyright (c) 2020, 2024, Red Hat Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2020, Red Hat Inc. All rights reserved.
 * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```
AFAIU, we don't update 3rd party copyright notices.